NitroxModel/DataStructures/FakeScheduledGoal.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using ProtoBufNet;
+
+namespace NitroxModel.DataStructures
+{
+    [Serializable]
+    [ProtoContract]
+    public class FakeScheduledGoal
```

Comment:
I would use "NitroxScheduledGoal" because "Fake" can be misleading, also it would be in line with other classes which were facing the naming problem.